docs/sphinx-docs/build_sphinx.py

```diff
@@ -166,35 +160,13 @@ def retrieve_user_docs():
 
 def retrieve_sasdata_docs():
     """
-        Copies select files from the bumps documentation into fitting perspective
+    Copies the sasdata documentation tree into sasview.
     """
     print("=== Sasdata Docs ===")
     copy_tree(SASDATA_DEV_SOURCE, SASDATA_DEV_TARGET)
     copy_tree(SASDATA_GUIDE_SOURCE, SASDATA_GUIDE_TARGET)
 
 
-def retrieve_bumps_docs():
-    """
-    Copies select files from the bumps documentation into fitting perspective
-    """
-    if exists(BUMPS_SOURCE):
-        print("=== Retrieve BUMPS Docs ===")
-        filenames = glob(joinpath(BUMPS_SOURCE, "dream-*.png"))
```

Comment:
The two dream images, `dream_complete.png` and `dream_incomplete.png` are not in the sasview repo. Moving these to the expected location is the last thing needed. Otherwise, this is ready to go.

Reply:
They are there already: https://github.com/SasView/sasview/tree/2673-bumps-docs/src/sas/qtgui/Perspectives/Fitting/media

How is it failing?

Note that the files have dash separators, not underscores.